root/static/scripts/common/components/TagEditor.js

```diff
@@ -451,15 +467,20 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
         const filteredTerms: $ReadOnlyArray<string> =
           sortByNumber(
             $.ui.autocomplete.filter(
-              self.genreNames.filter(x => !previousTerms.has(x)),
+              self.genreOptions.filter(x => !previousTerms.has(x.value)),
               last,
             ).sort(),
-            x => x.startsWith(last) ? 0 : 1,
+            x => x.value.startsWith(last) ? 0 : 1,
           );
 
         response(filteredTerms);
       },
-    });
+    }).data('ui-autocomplete')._renderItem = function (ul, item) {
+      return $('<li></li>')
+        .data('item.autocomplete', item)
```

Comment:
It appears to work without this line, and I checked the jQuery UI source and couldn't find any use of it. Odd.
```suggestion
```

Reply:
Huh. I guess it might be some sort of line that has gone through stackoverflow comments for years without anyone checking whether it was needed. It does work without, made the change.